linera-service/src/cli_wrappers/wallet.rs

```diff
@@ -412,7 +408,7 @@ impl ClientWrapper {
         module_id: &ModuleId<Abi, Parameters, InstantiationArgument>,
         parameters: &Parameters,
         argument: &InstantiationArgument,
-        required_application_ids: &[UserApplicationId],
+        required_application_ids: &[MultiAddress],
```

Comment:
Maybe this should just be called `required_applications` now everywhere.

In general, a variable of type `MultiAddress` should probably be called `app[lication]` or `app[lication]_addr[ess]`.

Reply:
Yes, there's a lot of renames still to do.

Reply:
This is another instance where using addresses looks imprecise.